tools/generate_hotkeys.py

```diff
@@ -12,28 +21,51 @@
     }
     for category in HELP_CATEGORIES.keys()
 }
-OUTPUT_FILE = Path(__file__).resolve().parent.parent / "docs" / "hotkeys.md"
-SCRIPT_NAME = PurePath(__file__).name
+OUTPUT_FILE = ROOT_DIRECTORY / "docs" / "hotkeys.md"
+SCRIPT_NAME = pathlib.PurePath(__file__).name
 
 with open(OUTPUT_FILE, "w") as mdFile:
     mdFile.write(
         f"<!--- Generated automatically by tools/{SCRIPT_NAME} -->\n"
         "<!--- Do not modify -->\n\n# Hot Keys\n"
     )
+    # keys_list = [item["keys"] for item in KEY_BINDINGS.values()]
+    # print("keys_list -", keys_list)
     for action in categories.keys():
         mdFile.write(
             f"## {HELP_CATEGORIES[action]}\n"
             "|Command|Key Combination|\n"
             "| :--- | :---: |\n"
         )
+        keys_list = []
+        for help_text, key_combinations_list in categories[action].items():
+            keys_list.append(key_combinations_list)
```

Comment:
Continuing the previous comment, if `for key in keys_list` can be pulled out of the for loop then I suppose the above can also be combined into the for loop below?

Reply:
I meant to say, if we can move duplicate checking to the end, then we can do everything in 1 for loop itself.